sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -55,6 +85,14 @@ def load(
     :paramtype trim_prefixes: Optional[List[str]]
     :keyword key_vault_options: Options for resolving Key Vault references
     :paramtype key_vault_options: ~azure.appconfiguration.provider.AzureAppConfigurationKeyVaultOptions
+    :keyword refresh_all: One or more settings whos modification will trigger a full refresh after a fixed interval.
```

Comment:
Would `invalidate_all` (or something similar) be a better name? Or maybe `trigger_refresh`?

Reply:
Yes I also thought "trigger" was a good verb here. I think `trigger_refresh` alone seems to be lacking, but not so much so that some docs wouldn't fill in the clarification gaps without needing to find a more descriptive name. 
So I'll sub in `trigger_refresh` for now and keep thinking on it.